lib/records/ccda.rb

```diff
@@ -126,7 +125,7 @@ def self.careplans(plan, ccda_record)
             'codes' => CAREPLAN_LOOKUP[entry][:codes],
             'description' => CAREPLAN_LOOKUP[entry][:description],
             'start_time' => time.to_i,
-            'reason' => COND_LOOKUP[plan['reason']]
+            'reason' => COND_LOOKUP[plan['reasons'][0]]
```

Comment:
We're potentially throwing away data here (and in line 148).  Does C-CDA just not support multiple reasons?